x/ccv/provider/keeper/genesis.go

```diff
@@ -33,7 +33,8 @@ func (k Keeper) InitGenesis(ctx sdk.Context, genState *types.GenesisState) {
 
 func (k Keeper) ExportGenesis(ctx sdk.Context) *types.GenesisState {
 	store := ctx.KVStore(k.storeKey)
-	iterator := sdk.KVStorePrefixIterator(store, []byte(providertypes.ChannelToChainKeyPrefix+"/"))
+	keyPrefix := string(types.ChannelToChainPrefix())
+	iterator := sdk.KVStorePrefixIterator(store, []byte(keyPrefix))
```

Comment:
It's a bit weird that we convert from []byte to string and then back to []byte.

Reply:
nice catch, this and other cleanups are addressed in https://github.com/cosmos/interchain-security/pull/242/commits/c944ad13197ce410e4fdb861f5f413e201dd162e